app/helpers/job_invocations_chart_helper.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal:true
+
+module JobInvocationsChartHelper
+  def job_invocation_chart(invocation)
+    donut_chart('#status_chart',
+                job_invocation_data(invocation)[:columns],
+                job_invocation_data(invocation)[:groups])
+  end
+
+  def job_invocation_data(invocation)
+    success = @job_invocation.progress_report[:success]
```

Comment:
I guess we should use argument `invocation`, not instance variable `@job_invocation` here. Although it doesn't seem to have an effect at the final state.